doc/source/cluster/cluster_under_construction/ray-clusters-on-vms/user-guides/launching-clusters/on-premises.md

```diff
@@ -0,0 +1,116 @@
+# Launching On-Premises Cluster
+
+This docs describes how to set up an on-premises Ray cluster, i.e., to run Ray on bare metal machines, or in a private cloud. We provide two ways to start an on-premises cluster.
+
+* You can **manually set up** the Ray cluster by installing the Ray package and starting the Ray processes on each node. 
+* Alternatively, if you know all the nodes in advance and have ssh access to them, you should start the ray cluster using **cluster-launcher**.
```

Comment:
`s/you should start the ray cluster using **cluster-launcher**/you should start the Ray cluster using the **cluster launcher**/g`

Thoughts on linking to the subsection instead of `**bolding**` them? IMO this makes it easier to traverse the document without losing any meaning, since they'll still be emphasized.